services/backend/src/routes/connectivity.js

```diff
@@ -114,6 +114,43 @@ module.exports = (errorHandler, connectivityService) => {
       });
   }));
 
+
+  /**
+   * @apiGroup s_pa_connectivity
+   * @api {get} /connectivity/timeline TimelineByCategory
+   * @apiName TimelineByCategory
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Values of protected and protected connected of all years for a given area
+   *
+   * @apiParam (Query params) {String|Number} areaType area type
+   * @apiParam (Query params) {String} areaId area id
+   * @apiParam (Query params) {String} category category of connectivity index
+   *
+   * @apiSuccess {Object[]} result
+   * @apiSuccess {String} result.id PA connectivity category
```

Comment:
id doesn't exist in the mockup, you are missing 2 properties and, as said in the mockup, i think this endpoint should return an object, not an array

Reply:
fixed, id was replaced by key.